docs/sources/getting-started/getting-started-prometheus.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Getting started with Grafana and Prometheus"
+description = "Guide for getting started with Grafana"
+keywords = ["grafana", "intro", "guide", "started"]
+type = "docs"
+aliases = ["/docs/grafana/latest/guides/gettingstarted","/docs/grafana/latest/guides/getting_started"]
+[menu.docs]
+identifier = "getting_started-grafana-prometheus"
+parent = "guides"
+weight = 300
++++
+
+# Getting Started with Grafana and Prometheus
+
+Prometheus is an open-source systems monitoring and alerting toolkit for which Grafana includes built-in support. This topic walks you through the steps to create a series of dashboards in Grafana displaying system metrics of a host monitored by Prometheus.
+
+You can also configure your Grafana instance in the cloud to display system metrics. For more information, refer to the [Grafana Cloud](https://grafana.com/docs/grafana-cloud/) documentation.
+
+## Step 1: Install Grafana and build your first dashboard
+
+Use the instructions in [Getting started with Grafana]({{< relref "getting-started.md" >}}) to:
+- Install Grafana
+- Login to Grafana
+- Create your first dashboard
+
+## Step 2: Download Prometheus and node exporter
+
+Prometheus, like Grafana, can be installed on many different operating systems. Refer to the [Prometheus download page](https://prometheus.io/download/) which lists all stable versions of Prometheus components. Download the following components:
+ - [prometheus](https://prometheus.io/download/#prometheus)
+ - [node_exporter](https://prometheus.io/download/#node_exporter)
+
+## Step 3: Install Prometheus node_exporter
+
+Prometheus Node Exporter is a widely used tool that exposes system metrics. Install node_exporter on all hosts you want to monitor. For instructions on how to install node_exporter, refer to the [Installing and running the Node Exporter](https://prometheus.io/docs/guides/node-exporter/#installing-and-running-the-node-exporter) section in Prometheus documentation.
+
+>Note: The instructions in the referenced topic is intended for Linux users. You may have to alter the instructions slightly depending on your operating system.
+
+## Step 4: Install and configure Prometheus
+
+1. Install Prometheus following instructions in the [Installation](https://prometheus.io/docs/prometheus/latest/installation/) topic in Prometheus documentation.
+
+1. Next, configure Prometheus to monitor the hosts where you installed node_exporter. In order to do this, configure prometheus.yaml file located in Prometheus's home directory. Here is an example of the code you will need to add.
```

Comment:
```suggestion
1. Configure Prometheus to monitor the hosts where you installed node_exporter. To do this, configure prometheus.yaml in Prometheus's home directory. Here's an example of the code you need to add.
```

Reply:
```suggestion 
1. Configure Prometheus to monitor the hosts where you installed node_exporter. To do this, modify Prometheus's YAML configuration file. Prometheus's configuration file is often called `prometheus.yml` and can be found in either `/etc/prometheus/prometheus.yml` or the directory where you've installed Prometheus. Here's an example of the code you need to add.
```

My rewrite is not very good either, but what I want to convey is that _home directory_ for UNIX environments refers specifically to the directory in `/etc/passwd` which is assigned to the `prometheus` user. On a typical Debian/Ubuntu installation, this'd be `/var/lib/prometheus`, but the configuration file lives in `/etc/prometheus/prometheus.yml`. The second instruction (where I replace "home directory" to "where you've installed Prometheus") should be more Windows-y, but I'm actually not sure. It is true for when you compile Prometheus and just run it with `./prometheus` on UNIX.

Ultimately, it's controlled by the `--config.file` flag, but since that's often not seen by the user there's no easy way to tell them what to look for 🙁

Reply:
IIRC the default is to use the file `prometheus.yml` (_not_ `prometheus.yaml`) in the _current work directory_.

Prometheus itself has no notion of checking the `/etc` directory or any home directory as listed in `/etc/passwd`.

Reply:
Yes, that's my understanding as well. I used "the directory where you've installed Prometheus" in place of cwd because Windows and because I couldn't think of the correct term. The `/etc` directory part came from the Docker instructions and also Debian's packaging (not that I think that we should use third-party packagers as an example of how to correctly set something up).

https://github.com/prometheus/prometheus/blob/master/Dockerfile#L26
https://packages.debian.org/sid/amd64/prometheus/filelist

Maybe 👇  would be more correct?

```suggestion
1. Configure Prometheus to monitor the hosts where you installed node_exporter. To do this, modify Prometheus's configuration file. By default Prometheus looks for the file `prometheus.yml` in the current working directory. Some Prometheus installers changes this to `/etc/prometheus/prometheus.yml`. Here's an example of the configuration you need to add.
```

Reply:
I think it's correct this way. Perhaps mention the command line flag, which is also used in the example below then, e.g.

"… By default Prometheus looks for the file `prometheus.yml` in the current working directory. This behavior can be changed via the `--config.file` command line flag. For example, some Prometheus installers use it to set the config file to `/etc/prometheus/prometheus.yml`. …"

Reply:
Thank you @sakjur and @beorn7, I have updated the instructions as per your suggestion.